src/Apps/CollectorProfile/Routes/Saves2/Components/__tests__/SavesArtworks.jest.tsx

```diff
@@ -21,15 +21,23 @@ const { renderWithRelay } = setupTestWrapperTL<SavesArtworks_Test_Query>({
       return null
     }
 
-    return <SavesArtworksRefetchContainer collection={props.me.collection} />
+    return (
+      <SavesArtworksRefetchContainer
+        collection={props.me.collection}
+        defaultSaves={props.defaultSaves!}
+      />
+    )
   },
   query: graphql`
     query SavesArtworks_Test_Query @relay_test_operation {
       me {
-        collection(id: "collectionID") {
+        collection: collection(id: "collectionID") {
           ...SavesArtworks_collection
         }
       }
+      defaultSaves: me {
+        ...SavesArtworks_me
+      }
```

Comment:
now that I changed the structure the tests fail because the structure changed, will fix it after a first feedback round